Project/Galagram/ViewModel/Commands/Admin/GoBackCommand.cs

```diff
@@ -30,6 +30,11 @@ public override void Execute(object parameter)
         {
             Core.Logger.GetLogger.LogAsync(Core.LogMode.Debug, $"Exetute {nameof(GoBackCommand)}");
 
+            // You don't have to change entity if you want to go back.
+            // The entity is detached after it has been created and it isn't yet in database.
+            if (parameter != null && DataAccess.Context.UnitOfWork.Instance.AppContext.Entry(parameter).State != System.Data.Entity.EntityState.Detached)
+                DataAccess.Context.UnitOfWork.Instance.AppContext.Entry(parameter).State = System.Data.Entity.EntityState.Unchanged;
```

Comment:
* curve brackets is missing
* Logger is missing
* also seems to be not a right place. Why **GoBackCommand** knows about entity?

Reply:
I thinck this could be done in **SingleItemViewModelBase** constructors. but with **detached** not **unchanged**. So you do not event need to pass argument in every View